.github/ISSUE_TEMPLATE/Bug Report.yaml

```
@@ -0,0 +1,142 @@
+      label: Precise OCCT Version or Branch
+      description: "Specify the precise OCCT version, branch, or SSH (optional)"
+  - type: input
+    id: environment
+    attributes:
+      label: Environment
+      description: "Details about your environment (OS, compiler, etc.)"
+  - type: textarea
+    id: additional_files
+    attributes:
+      label: Additional Files
+      description: "Attach any additional files that may help in reproducing the issue"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)